resources/views/components/forms/inputs/flat-pickr.blade.php

```diff
@@ -1,6 +1,6 @@
 <input
-    x-data
-    x-init="flatpickr($el, {{ $jsonOptions() }})"
+    x-data="{picker:null}"
+    x-on:click="picker=picker||flatpickr($el, {{ $jsonOptions() }}),picker.open()"
```

Comment:
Instead of `:click`, could we not do `:mouseenter` instead. If you click on the input, there could potentially be a delay between you clicking and the flatpickr instance being initialised.

With the `:mouseenter`, we'd essentially be preloading the instance instead and then Flatpickr can control the `open` / `close` part instead. What do you think?

Reply:
It makes total sense to me. We have to consider that maybe, flatpickr isn't instantiated fast enough. so there is nothing that can react to the click. Have no idea how to solve this without blocking the UI